app/views/job_runs/discovery_report_summary.html.erb

```diff
@@ -0,0 +1,63 @@
+<%= turbo_frame_tag 'discovery_report' do %>
+  <hr/>
+  <h5>Results summary</h5>
+  <table class="table">
+    <thead>
+      <tr class="text-center">
+        <th scope="col">Num Objects</th>
+        <th scope="col">Num with Errors</th>
+        <th scope="col">Total Size</th>
+        <th scope="col">Started</th>
+        <th scope="col">Run Time</th>
+        <th scope="col">Mime types</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td class="text-center"><%= @discovery_report['rows'].size %></td>
+        <td class="text-center"><%= @discovery_report['summary']['objects_with_error'] %></td>
+        <td class="text-center"><%= number_to_human_size(@discovery_report['summary']['total_size'], precision: 2) %></td>
+        <td class="text-center"><%= @discovery_report['summary']['start_time'].to_time.strftime('%-m/%-d/%y %T') %></td>
+        <% if @discovery_report['summary']['end_time'] && @discovery_report['summary']['start_time'] # not all older discovery reports have both times available %>
+          <td class="text-center"><%= distance_of_time_in_words(@discovery_report['summary']['end_time'].to_time - @discovery_report['summary']['start_time'].to_time) %></td>
+        <% else %>
+           <td class="text-center">unknown</td>
+        <% end %>
+        <td><%= render partial: 'mime_type_summary', locals: {mimetypes: @discovery_report['summary']['mimetypes']}%></td>
```

Comment:
odd that `<%=` in these lines appear as comments ... I assume the lines display properly

Reply:
yes, it works fine in the browser